templates/field/field--entity-reference-revisions--paragraph--field-oe-paragraphs.html.twig

```diff
@@ -0,0 +1,9 @@
+{#
+/**
+ * @file
+ * Theme override for the entity reference revisions field wich allowed to use paragraphs.
+ *
+ * @see ./templates/field/field--bare.html.twig
+ */
+#}
+{%- include 'field--bare.html.twig' -%}
```

Comment:
Not sure if this is not too broad, maybe we should have the following suggestions and pick the post restrictive one:

We can use these to complete the chain prefixed by TARGET_TYPE: :point_down: 
```
field__entity_reference_revisions__TARGET_TYPE
field__entity_reference_revisions__TARGET_TYPE__TARGET_BUNDLE
field__entity_reference_revisions__TARGET_TYPE__TARGET_BUNDLE__FIELD_NAME
```
But then we also add those that have precedence over those above:
```
field__entity_reference_revisions__PARENT_ENTITY_TYPE
field__entity_reference_revisions__PARENT_ENTITY_TYPE__PARENT_ENTITY_BUNDLE
field__entity_reference_revisions__PARENT_ENTITY_TYPE__PARENT_ENTITY_BUNDLE__FIELD_NAME
```

And pick up the last one: 

```
field__entity_reference_revisions__PARENT_ENTITY_TYPE__PARENT_ENTITY_BUNDLE__FIELD_NAME
```

Does it make sense?

Reply:
I understand your point.
I tested all the components and configurations, and it seems to work.